server/index.js

```diff
@@ -19,11 +24,32 @@ app.get('/houses/*', (req, res) => {
           res.send(rows); 
         })
         .then((res) => {
-          console.log(res);
+          //console.log(res);
+          conn.end();
+        })
+        .catch(err => {
+          //console.log(`Error: ${err}`);
+          conn.end();
+        })
+    }).catch(err => {
+      console.log(`Not connected to MariaDB: ${err}`);
+    });
+  });
+
+  app.get('/test', (req, res) => {
+    let houseID = getRandomInt(1e7);
+    House.getConnection()
+    .then(conn => {
+      conn.query(`SELECT * FROM homes WHERE homeID=${houseID}`)
```

Comment:
This form of querying leaves you open to **SQL Injection attacks**! _Never_ use raw template strings to query your database. Either find some way of sanitizing your inputs before placing them in a query string, or use a prepared statement (which looks like `SELECT * FROM homes WHERE homeID=$1` and then you find some way to pass `[ houseID]` for `$1` probably by using `conn.query`'s optional arguments... or maybe some other method like `conn.execPreparedStatement` or something